angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html

```diff
@@ -0,0 +1,251 @@
+<main class="container-fluid-padding">
+  <form *ngIf="!loading" [formGroup]="myForm" novalidate>
+    <section>
+      <div class="d-flex heading-bar">
+        <div>
+          <h1 id="name" class="m-0">
+            {{ (isEditing && collection && collection.name) || (!isEditing && myForm.get('collectionName').value) }}
+          </h1>
+        </div>
+      </div>
+    </section>
+
+    <section class="mb-3 pb-3 border-bottom">
+      <div class="d-flex justify-content-between">
+        <div class="col-6 col-lg-8 col-xl-9">
+          <div>
+            <h2 class="border-0 mb-0">Edit Collection</h2>
+          </div>
+          <div class="mb-3">
+            <span class="grey-subtext">{{ lastEditedSubText }}</span>
+          </div>
+          <div>
+            <mat-slide-toggle
+              name="collectionPublish"
+              id="collectionPublish"
+              class="publish-toggle"
+              [ngClass]="myForm.get('collectionPublish').value ? 'published' : 'unpublished'"
+              formControlName="collectionPublish"
+              (change)="togglePublish($event)"
+              [checked]="myForm.get('collectionPublish').value"
+              title="{{ myForm.get('collectionPublish').value ? 'Unpublish collection' : 'Publish collection' }}"
+              ngDefaultControl
+            >
+              {{ (myForm.get('collectionPublish').value && 'Published') || 'Unpublished' }}
+            </mat-slide-toggle>
+          </div>
+        </div>
+        <div class="col-6 col-lg-4 col-xl-3">
+          <div class="d-flex justify-content-end mb-1">
+            <button type="button" class="btn btn-link" (click)="cancel()" title="Cancel">Cancel</button>
+            <button class="btn btn-primary" type="submit" (click)="submit()" title="Update">
```

Comment:
Button should say "save" when creating a new record?

Reply:
Yup, will fix